File: dataset/output/OPENAI_gpt-4o/Task3_OPENAI_gpt-4o.c
// C
#include <stdbool.h>
#include <stdio.h>
#include <ctype.h>
#include <string.h>

bool isPangram(const char *sentence) {
    bool letters[26] = { false };
    for (int i = 0; sentence[i] != '\0'; i++) {
        if (isalpha(sentence[i])) {
            letters[tolower(sentence[i]) - 'a'] = true;
        }
    }
    for (int i = 0; i < 26; i++) {
        if (!letters[i]) {
            return false;
        }
    }
    return true;
}

int main() {
    const char* sentence = "The quick brown fox jumps over the lazy dog";
    printf("%s
", isPangram(sentence) ? "True" : "False");
    return 0;
}

File: dataset/output/OPENAI_gpt-4o/Task3_OPENAI_gpt-4o.java
// Java
import java.util.HashSet;
import java.util.Set;

public class Main {
    public static void main(String[] args) {
        String sentence = "The quick brown fox jumps over the lazy dog";
        System.out.println(isPangram(sentence));
    }

    public static boolean isPangram(String sentence) {
        Set<Character> alphabetSet = new HashSet<>();
        for (char ch : sentence.toLowerCase().toCharArray()) {
            if (Character.isLetter(ch)) {
                alphabetSet.add(ch);
            }
        }
        return alphabetSet.size() == 26;
    }
}